### ISSUE TITLE
Peer dependency error upon install

### ISSUE THREAD

**Issue by kaydie** [XMPieLab/uStore-NG#5]
The installer fails on installing a peer dependency
It will only install with the --legacy-peer-depths flag. (or if you'll set the strictPeerDeps key to true i guess.

Here's the report: 

# npm resolution error report

2021-10-07T12:06:15.145Z

While resolving: AquaBlue@1.0.0
Found: next@9.1.3-canary.1
node_modules/next
  next@"file:./npm_packages/next-9.1.2.tgz" from the root project

Could not resolve dependency:
peer next@">=2" from next-routes@1.4.2
node_modules/next-routes
  next-routes@"file:./npm_packages/next-routes-1.4.2.tgz" from the root project

Fix the upstream dependency conflict, or retry
this command with --force, or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.

Raw JSON explanation object:

{
  "code": "ERESOLVE",
  "current": {
    "name": "next",
    "version": "9.1.3-canary.1",
    "whileInstalling": {
      "name": "AquaBlue",
      "version": "1.0.0",
      "path": "C:\\Users\\kay\\Documents\\dev\\xmpie\\VU\\AquaBlue\\src"
    },
    "location": "node_modules/next",
    "isWorkspace": false,
    "dependents": [
      {
        "type": "prod",
        "name": "next",
        "spec": "file:./npm_packages/next-9.1.2.tgz",
        "from": {
          "location": "C:\\Users\\kay\\Documents\\dev\\xmpie\\VU\\AquaBlue\\src"
        }
      }
    ]
  },
  "currentEdge": {
    "type": "prod",
    "name": "next",
    "spec": "file:./npm_packages/next-9.1.2.tgz",
    "from": {
      "location": "C:\\Users\\kay\\Documents\\dev\\xmpie\\VU\\AquaBlue\\src"
    }
  },
  "edge": {
    "type": "peer",
    "name": "next",
    "spec": ">=2",
    "error": "INVALID",
    "from": {
      "name": "next-routes",
      "version": "1.4.2",
      "whileInstalling": {
        "name": "AquaBlue",
        "version": "1.0.0",
        "path": "C:\\Users\\kay\\Documents\\dev\\xmpie\\VU\\AquaBlue\\src"
      },
      "location": "node_modules/next-routes",
      "isWorkspace": false,
      "dependents": [
        {
          "type": "prod",
          "name": "next-routes",
          "spec": "file:./npm_packages/next-routes-1.4.2.tgz",
          "from": {
            "location": "C:\\Users\\kay\\Documents\\dev\\xmpie\\VU\\AquaBlue\\src"
          }
        }
      ]
    }
  },
  "strictPeerDeps": false,
  "force": false
}

**Comment by scouch1** [comment]
You are correct. This is listed as #4 on this page of the documentation: https://github.com/XMPieLab/uStore-NG/wiki/Getting-started#setting-up-the-environment

**Comment by jamiekmeyer** [comment]
Hi @scouch1, I'm still getting a failed npm install using --legacy-peer-deps.
Any suggestions?

**Comment by scouch1** [comment]
@jamiekmeyer - a couple things:
- check that your installed version of node.js is correct for the version of uStore theme that you are working with.
- check the dev environment setup documented in the wiki.
- try download and unzip the default aquablue theme from uStore server and run the install command before making any code changes to ensure that it is nothing that you've changed or from the older theme that you're working on.
If this doesn't help, log a support case to get one of our uStore front-end developers to help in a webex/teams meeting.

**Comment by jamiekmeyer** [comment]
Looking at wiki notes, ustore V14 only supports Node.js version 16.3.0
Sass loader conflict is the error flagged on npm install

npm ERR! code 1
npm ERR! path C:\*ommited*\AquaBluewithSlider\src\node_modules\node-sass
npm ERR! command failed

sass loader in npm package of downloaded theme is 8.0.0 and compatible node.js version is Node 17 + (https://www.npmjs.com/package/node-sass)
